proposed/2021/NuGetOrgDetailsPageRedesign.md

```diff
@@ -0,0 +1,140 @@
+
+
+### Success: How do we know if we’ve solved this problem?
+- By HaTS survey results and customer interview replies  
+- By measuring rates at which people click on sections/tabs on the package details page  
+- By measuring changes in the NuGet.org traffic for the details page of our top 100 popular packages
+- By gauging excitement/disappointment after blogging about this feature & analyzing sentiment on Twitter/GitHub/DevCom/etc.
+
+
+### Audience: Who are we building for?
+
+We are building this for the .NET developer who is browsing & making decisions about including a NuGet package. They want to:
+-	Find information they need quickly and easily
+-	Use higher quality packages
+-	Use more secure packages
+
+We are also building this for the .NET developer who is creating & publishing NuGet packages for public use. They want to:
+-	Display relevant information about their package
+-	Increase usage of their package
+
+## Goals and Non-Goals
+
+### Header
+
+![](../../meta/resources/NuGetPackageDetails/headernew.png)
+
+Information that needs to be seen at a glance such as package ID, description, and eventually, TFM badges.
+
+**Requirements**        
+- Title   
+- Icon  
+    - Hidden for packages without icon
+- Version   
+- Checkmark to indicate reserved prefix     
+  - includes link to prefix reservation docs
+- Installation instructions
+  - Dropdown for choosing variant
+  - Sticks to chosen variant for all future browsing
+  - Additional, detailed installation instructions for new users in Installation Tab
+- Vulnerability and other critical warning banners 
+  - will be shown at very top of page 
+  - catches immediate attention
+
+**Non-Requirements**
+- Target Framework badges
+  - It's coming, just in a separate proposal!
+
+
+### Sidebar
+
+Additional helpful information such as package installation instructions, statistics, external links. 
+
+![](meta/resources/NuGetPackageDetails/sidebar_final.png)
+
+**Requirements:**
+- Download statistics 
+  - Total downloads
+  - Downloads of current version
+  - Average downloads per day
+- External links
+  - Personal site
+  - Source repository
+  - License
+- Download package
+- Open in FuGet
+- Owners
+  - Contact owners
+- Tags 
+- Report
+  - Leads to a report page where users can choose from reporting copyright infringement, security vulnerabilities, inappropriate content, and others (in the works)
+- Copyright
+
+**Non-Requirements:**
+- Authors
+    - Eliminated from page due to hearing confusion from community surrounding authors vs. owners
+      - There are 6,686 package IDs with "Microsoft" as its author. 
+      - Of these, 23.2% are not actually owned by Microsoft. These packages have 330M total downloads. (Note that one package ID may have many package versions)
+
+
+
+### Body
+
+Tabs with titles that clearly convey what information is in them.
+
+**Requirements:**       
+- README Tab
```

Comment:
Consider changing the capitalization to "Readme". All caps looks a little loud and distracting compared to the other tab labels. While "README" is frequently used in documentation, both Pub.Dev and NPM use "Readme" in tab labels, probably for the reason stated above 😊

Reply:
Great point about the difference in usage within documentation vs. tab labels! For easy reference, this is what pub.dev and NPM looks like:

![image](https://user-images.githubusercontent.com/23298667/118849517-0e046280-b88d-11eb-8ccf-4791f46a1743.png) 
![image](https://user-images.githubusercontent.com/23298667/118849573-19578e00-b88d-11eb-9bd4-0dd491c6fe53.png)

@loic-sharma  @sophiamfavro What do you two think?